.gitignore

```diff
@@ -4,6 +4,7 @@
 *.glob
 *.pb
 *.log
+*.sv
```

Comment:
This would exclude human written SystemVerilog files. Instead, in each directory, name the SystemVerilog files that need to be ignored.